src/game/objects/Heart.js

```diff
@@ -0,0 +1,29 @@
+import { AssetsManager } from '../assets';
+import { GameObject } from '../engine';
+import { Vector } from '../shared';
+
+class Heart extends GameObject {
+  #sprite = AssetsManager.heart;
+
+  /**
+   * @param {Object} props
+   * @param {Vector} props.initialPos
+   */
+  onActivate({ initialPos }) {
+    this.#sprite = AssetsManager.heart;
```

Comment:
Tej linijki nie potrzebujesz, bo ustawiasz `this.#sprite` już w linii 6 na dokładnie to samo.